src/main/java/page_object/HomePage.java

```diff
@@ -1,21 +1,13 @@
 package page_object;
 
-import helper.DriverHelper;
-import helper.ElementHelper;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
+import com.logigear.control.common.imp.Label;
 
 public class HomePage extends BasePage {
     // Locator
-    private final By lblWelcomeContentAtTop = By.cssSelector("div#content h1");
-
-    // Element
-    private WebElement getLblWelcomeContentAtTop() {
-        return DriverHelper.getDriver().findElement(lblWelcomeContentAtTop);
-    }
+    private final Label lblWelcomeContentAtTop = new Label("css=div#content h1");
```

Comment:
if you have only 1 Welcome header then just use lblWelcomeContent, no need to add location, please apply it for all pages

Reply:
Fixed. Please check again,